packages/cli/src/Init.ts

```diff
@@ -344,26 +356,51 @@ export class Init implements Command {
         console.log(`Successfully authenticated as ${bold(authenticationResult.email)}.`)
       }
 
+      if (args['--prompt'] || args['--vibe']) {
+        const prompt = args['--prompt'] || args['--vibe'] || ''
+        const spinner = ora(`Generating a Prisma Schema based on your description ${bold(prompt)} ...`).start()
+
+        try {
+          ;({ generatedSchema, generatedName } = (await (
+            await fetch(`https://prisma-generate-server.prisma.workers.dev/`, {
+              method: 'POST',
+              body: JSON.stringify({
+                description: prompt,
+              }),
+            })
+          ).json()) as { generatedSchema: string; generatedName: string })
```

Comment:
Could we validate it using Effect Schema or zod instead of just asserting the type? We generally use Effect Schema in our code, so it's probably better to stick to using it here as well for consistency, and keep zod isolated to `@modelcontextprotocol/sdk` integration.